### PR TITLE
home page links and buttons validation test

### PR DESCRIPTION
**Pull request by NataliaSantosB**
# Description

Home page all links and buttons validation test

https://app.qase.io/project/ACQ?view=1&suite=5&case=33&previewMode=modal
https://app.qase.io/project/ACQ?view=1&suite=5&case=34&previewMode=modal
https://app.qase.io/project/ACQ?view=1&suite=5&case=35&previewMode=modal

# Checklist:

- [x] New tests

### CODE REVIEW

File: cypress/integration/ProdTests/Homefull.test.spec.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        const homePage = new HomePage();
+        homePage.getRecentBrand().invoke('text').then(text =>{
+            homePage.getRecentBrand().click()
+            cy.contains('h1', text).should('exist')
+
+        })
+
+    })
+
+    it.only ('Categories', () =>{
```
Comment (andreyAut) [inline cypress/integration/ProdTests/Homefull.test.spec.js:84]: Remember that "**only**" runs this test only. If you want to run the others, remember to remove the "only"

File: cypress/integration/ProdTests/Homefull.test.spec.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+ 
+        const homePage = new HomePage();
+        homePage.getCategoryPills().eq(1).invoke('text').then(text => {
+            homePage.getCategoryPills().eq(1).click()
+            //cy.contains('h1', text).should('exist')   //ESTA PEGANDO O NOME COM FORMATACAO
```
Comment (andreyAut) [inline cypress/integration/ProdTests/Homefull.test.spec.js:89]: this comment, I believe you can remove it from here.

File: cypress/integration/ProdTests/Homefull.test.spec.js (hunk 1 above)
Comment (NataliaSantosB) [reply cypress/integration/ProdTests/Homefull.test.spec.js:84]: Thank you Andrey. I will remove all

**Review by CaiqueCoelho — CHANGES_REQUESTED** [review]
Excellent Pull Request, I have left just minor improvements you can implement

File: cypress/integration/ProdTests/Homefull.test.spec.js (hunk 2 above)
Comment (NataliaSantosB) [reply cypress/integration/ProdTests/Homefull.test.spec.js]: Made the necessary adjustment to the code and it has already been removed.

File: cypress/integration/ProdTests/Homefull.test.spec.js [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+describe('Home Page tests', () => {
+
+    beforeEach(() => {
+        cy.visit('/')   
+    })
+    it('Header test', () =>{
+        
+        const homePage = new HomePage();
+        cy.contains('Lojista').should('have.attr','href').and('include', '/')
+        cy.contains('Fornecedor').should('have.attr','href').and('include', '/pages/fornecedor')
```
[the remark this answers is not in the document]
Comment (NataliaSantosB) [reply cypress/integration/ProdTests/Homefull.test.spec.js:15]: Made the necessary adjustment to the code and it has already been removed.

File: cypress/integration/ProdTests/Homefull.test.spec.js [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+        cy.contains('Contato').should('have.attr','href').and('include', '/pages/contato')
```
[the remark this answers is not in the document]
Comment (NataliaSantosB) [reply cypress/integration/ProdTests/Homefull.test.spec.js]: Made the necessary adjustment to the code and it has already been removed.

File: cypress/integration/ProdTests/Homefull.test.spec.js [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+        cy.contains('Painel Fornecedor').should('have.attr','href').and('include', 'https://fornecedor.inventa.app.br')
```
[the remark this answers is not in the document]
Comment (NataliaSantosB) [reply cypress/integration/ProdTests/Homefull.test.spec.js:16]: Made the necessary adjustment to the code and it has already been removed.

File: cypress/integration/ProdTests/Homefull.test.spec.js [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+        homePage.getPlayerYouTube().should('be.visible')
+
+    })
+
+    it ('Recent Brands', () =>{
+
+        const homePage = new HomePage();
+        homePage.getRecentBrand().invoke('text').then(text =>{
+            homePage.getRecentBrand().click()
+            cy.contains('h1', text).should('exist')
```
[the remark this answers is not in the document]
Comment (NataliaSantosB) [reply cypress/integration/ProdTests/Homefull.test.spec.js:64]: Made the necessary adjustment in the code

**Review by andreyAut — APPROVED** [review]

**Review by CaiqueCoelho — APPROVED** [review]

**Pull request merged**